packages/astro/src/runtime/server/index.ts

```diff
@@ -166,6 +166,10 @@ function formatList(values: string[]): string {
 	return `${values.slice(0, -1).join(', ')} or ${values[values.length - 1]}`;
 }
 
+const rendererAliases = new Map([
+	['solid', 'solid-js']
+])
```

Comment:
Unrelated bug caught by update to `e2e` tests. We should support both `client:only="solid"` and `client:only="solid-js"`